### ISSUE TITLE
"reset new books" and "export raw data" buttons greyed out in 0.2.9 for Chrome

### ISSUE THREAD

**Issue by boisjere** [joonaspaakko/audible-library-extractor#129]
As the title says... 

I want a CSV version of my library to rework my Collections structure. So I need an editable spreadsheet.

The Console shows 28 issues (all either net::ERR_BLOCKED_BY_CLIENT or Uncaught Reference Error: ttd_dom_ready is not defined).

![screenshot_655](https://github.com/joonaspaakko/audible-library-extractor/assets/87881017/242b9736-ffdd-4722-977a-d1c491381bad)

Since it's "Blocked by Client", does that mean it's on my side, like anti-virus software or a VPN?

Some of the Network Headers are red, if that matters. Also, I have some book cover images in there, even though I clicked "remove all extracted data" several times. These all seem to be the cover images of new books, added since my last extraction. But my spreadsheet from the last extraction got corrupted, so I need to start from scratch again, and export the whole thing.

![screenshot_657](https://github.com/joonaspaakko/audible-library-extractor/assets/87881017/a80c020c-24eb-4a44-bb12-4f5db935bf8e)

Timing data is below. (I don't really know what I'm doing, so I'm sharing anything that might be helpful!)

![screenshot_658](https://github.com/joonaspaakko/audible-library-extractor/assets/87881017/f85b9d30-1cbd-4dee-95b0-b94b3c9fbfca)

**Comment by joonaspaakko** [comment]
Both buttons should only be usable if you have extracted data, and since it sounds like you clicked the "remove all data" button, that should explain why.

I'm not seeing any errors that would directly link to the extension. You get there types of errors on the Audible website just sitting there doing nothing.

> But also, the "export raw data" button saves a JSON file for safe keeping, so you can use the "import raw data" button later if needed. It's not where you export a spreadsheet. But also in most situations export and import raw data buttons aren't needed because you can just do a new full extraction, it just takes a bit longer. It's really quite a niche feature that most people shouldn't need.

It sounds like you should [do a new extraction](https://joonaspaakko.gitbook.io/audible-library-extractor/how-to-use-the-extension) and then in the gallery [do a CSV export](https://joonaspaakko.gitbook.io/audible-library-extractor/gallery/csv-export).

> The reason for why the CSV export is in the gallery is because then you can leverage it's features for filtering and sorting the data before export. You can for example export csv from a specific sub page only. Maybe export CSV from a specific series. Maybe your library,  but first filter out a certain language or maybe plus catalog books. Or maybe sort it based on the length or something.

**Comment by joonaspaakko** [comment]
Closing for inactivity. You can still comment _(or open a new issue)_ if anything is unclear.
